requirements-test.txt

```diff
@@ -9,3 +9,4 @@ pytest-mock==1.10.0
 coveralls==1.5.1
 pytest-cov==2.6.1
 py-mini-racer==0.1.18
+attrs==19.1.0
```

Comment:
Fix 
```
[command]/bin/bash --noprofile --norc /home/vsts/work/_temp/8eb70c1f-fd00-43a7-bea0-aa872dac2bfe.sh
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.5.7/x64/bin/pytest", line 10, in <module>
    sys.exit(main())
  File "/opt/hostedtoolcache/Python/3.5.7/x64/lib/python3.5/site-packages/_pytest/config/__init__.py", line 61, in main
    config = _prepareconfig(args, plugins)
  File "/opt/hostedtoolcache/Python/3.5.7/x64/lib/python3.5/site-packages/_pytest/config/__init__.py", line 182, in _prepareconfig
    config = get_config()
  File "/opt/hostedtoolcache/Python/3.5.7/x64/lib/python3.5/site-packages/_pytest/config/__init__.py", line 156, in get_config
    pluginmanager.import_plugin(spec)
  File "/opt/hostedtoolcache/Python/3.5.7/x64/lib/python3.5/site-packages/_pytest/config/__init__.py", line 530, in import_plugin
    __import__(importspec)
  File "/opt/hostedtoolcache/Python/3.5.7/x64/lib/python3.5/site-packages/_pytest/tmpdir.py", line 25, in <module>
    class TempPathFactory(object):
  File "/opt/hostedtoolcache/Python/3.5.7/x64/lib/python3.5/site-packages/_pytest/tmpdir.py", line 35, in TempPathFactory
    lambda p: Path(os.path.abspath(six.text_type(p)))
TypeError: attrib() got an unexpected keyword argument 'convert'
```
error.

Refer to https://github.com/pytest-dev/pytest/issues/3280#issuecomment-538031871.